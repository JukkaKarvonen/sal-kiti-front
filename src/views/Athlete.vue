<template>
  <div class="athlete">
    <Athlete />
    <Results />
  </div>
</template>

<script>
/**
 * Athlete information and results view
 */
import Athlete from "@/components/AthleteInformation.vue";
import Results from "@/components/AthleteResults.vue";

export default {
  name: "AthleteView",
  components: {
    Athlete,
    Results
  },
  created() {
    document.title = this.$t("title.athlete") + " | " + this.$t("title.prefix");
  }
};
</script>
