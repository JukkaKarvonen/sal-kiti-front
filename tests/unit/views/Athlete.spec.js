import { shallowMount, createLocalVue } from "@vue/test-utils";
import Athlete from "@/views/Athlete.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("Athlete.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(Athlete, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
