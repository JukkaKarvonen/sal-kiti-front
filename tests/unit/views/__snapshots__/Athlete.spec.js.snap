// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Athlete.vue renders content 1`] = `
<div
  class="athlete"
>
  <athlete-stub />
   
  <results-stub />
</div>
`;
